CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if(USE_HIP_CPU)
 endif()
 
 # Setup VERSION
-set(VERSION_STRING "3.2.0")
+set(VERSION_STRING "3.2.2")
```

Comment:
Just wanted to double-check - should this be 3.2.1?

Reply:
I think it should be 3.2.2 - looking at the changelog for 6.2 https://github.com/ROCm/rocPRIM/blob/release/rocm-rel-6.2/CHANGELOG.md
We had an update for 6.2.1 which bumped it up to 3.2.1 (I forgot to do that earlier).  The gfx1151 cherry pick is for 6.2.4 which warrants another patch version bump.

Reply:
Ok, got it, thanks.